source/declarations.tex

```diff
@@ -3661,8 +3661,8 @@
 void g4(C3 auto);
 \end{codeblock}
 
```

Comment:
Maybe also delete these blank lines? We aren't really breaking any paragraphs here. (Also on line 3672.)

Reply:
fixed, thanks!